docs/API.md

```diff
@@ -6,6 +6,10 @@ These are best practices related to the current API used for App. This does not
 - There should be no client-side handling that is unique to any API call.
 - Data is pushed to the client and put straight into Onyx by low-level libraries.
 - Clients should be kept up-to-date with many small incremental changes to data.
+- Creating data needs to be optimistic on every connection (offline, slow 3G, etc), eg. `RequestMoney` or `SplitBill` should work without waiting for a server response.
```

Comment:
`When possible, creating data should be optimistic on every connection (offline, slow 3G, etc)...`?

Just saying because in some rare cases we don't do optimistic :P

Reply:
Oh what are those cases that we don't do optimistic when creating data? I think most of the cases will be optimistic with the client-generated ID. Also, I'm not sure about using the wording `When possible`, because that will open the possibility of not doing it when we should do it 😄